mentoring/public/js/questionnaire.js

```diff
@@ -99,14 +109,17 @@ function MCQBlock(runtime, element) {
                 var choiceTipsCloseDOM;
 
                 if (result.status === "correct" && choiceInputDOM.val() === result.submission) {
+                    choiceDOM.addClass('correct');
```

Comment:
I assume that the `correct` and `incorrect` classes are meant to be a toggle, where either one can be active but not both simultaneously. To ensure that, you probably want to always remove the other when you add one. For example:
```js
choiceDOM.removeClass("incorrect").addClass("correct");
```